### ISSUE TITLE
[Feature Request] Query events

### ISSUE THREAD

**Issue by lorepieri8** [bandprotocol/pylibra#3]
Do you plan on creating new api functions, like query events?

**Comment by sorawit** [comment]
Yes! I hope to be continuously adding more features over the coming days. But if there are any specific features that you want to prioritize, feel free to open a pull request. Thanks!
